Mathlib/LinearAlgebra/FreeModule/PID.lean

```diff
@@ -327,19 +331,19 @@ noncomputable def Submodule.basisOfPid {ι : Type _} [Finite ι] (b : Basis ι R
 theorem Submodule.basisOfPid_bot {ι : Type _} [Finite ι] (b : Basis ι R M) :
     Submodule.basisOfPid b ⊥ = ⟨0, Basis.empty _⟩ := by
   obtain ⟨n, b'⟩ := Submodule.basisOfPid b ⊥
-  let e : Fin n ≃ Fin 0 := b'.index_equiv (Basis.empty _ : Basis (Fin 0) R (⊥ : Submodule R M))
-  obtain rfl : n = 0 := by simpa using fintype.card_eq.mpr ⟨e⟩
+  let e : Fin n ≃ Fin 0 := b'.indexEquiv (Basis.empty _ : Basis (Fin 0) R (⊥ : Submodule R M))
+  obtain rfl : n = 0 := by simpa using Fintype.card_eq.mpr ⟨e⟩
   exact Sigma.eq rfl (Basis.eq_of_apply_eq <| finZeroElim)
 #align submodule.basis_of_pid_bot Submodule.basisOfPid_bot
 
 /-- A submodule inside a free `R`-submodule of finite rank is also a free `R`-module of finite rank,
 if `R` is a principal ideal domain.
 
-See also the stronger version `submodule.smith_normal_form_of_le`.
+See also the stronger version `Submodule.smithNormalFormOfLe`.
 -/
 noncomputable def Submodule.basisOfPidOfLe {ι : Type _} [Finite ι] {N O : Submodule R M}
```

Comment:
I think we've decided to capitalise as `LE` in names like this.

Reply:
I changed it and changed also `Submodule.basisOfPidOfLe` in the same way. It felt strange to change `Submodule.basisOfPidOfLeSpan` so I left it like that but I can change it too if you want.

Reply:
Yes please.